### DISCUSSION TITLE
CRUD service v7 encryption and views error

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#524] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.4

### Description

Hi, I created a collection with some encrypted fields and a view targeting that collection.
With crud-service v6.8.0 it works fine, but with crud-service v7.0.0 (and v7.0.4 also) there is an error during startup that says `MongoServerError: Creating a view is not supported with automatic encryption`

Do you experience something similar? 

I haven't found anything related in the changelog of the service, I think that is something related to mongojs or similar

**Comment by fredmaggiowski** [comment, marked as answer]
This problem also affects v6.10.2 (even though it has been fixed with v6.10.3

https://github.com/mia-platform/crud-service/compare/v6.10.2...v6.10.3

**Comment by albertotessarotto** [comment]
Hi @EdoCrippaOfficial , which MongoDB version are you using for your project?

**Reply by edocrippaofficial** [reply]
v6.0.16
